src/cromshell/utilities/cromshellconfig.py

```diff
@@ -23,6 +24,25 @@
 submission_file = None
 cromshell_config_options = None
 cromwell_server = None
+# Request defaults
+requests_connect_timeout = 5
+requests_verify_certs = True
```

Comment:
I'd strongly advocate for NOT including this not part of config, as this is most likely a temporary need (but is needed), so I'd do everything to make it NOT part of a normal config object.

Reply:
Currently certs toggle is only through the cli

Reply:
I'm thinking along the following lines given that somewhere down the line, there will be a function that generates a template config file for cromshell. So I could foresee some keys are appropriate to be included in that template, while others are not, and it may also need a complex design that reflect the json format.

Exactly how to design that is not clear to me yet, but it's nice to know that this is configurable only through CLI explicitly.